api/team.go

```diff
@@ -67,6 +67,8 @@ func createTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	app.SanitizeTeam(rteam, c.IsSystemAdmin())
```

Comment:
Should we be sanitizing the team object on creation? The creator will be the team admin so why sanitize out data?

Reply:
That's a good question and that actually got me a few times when testing since I would create the team and then those fields wouldn't be set. I was going with sanitizing whenever it's returned, but that's probably overkill. We probably don't even need to sanitize for team admins either

Reply:
I'll switch this over to not sanitize on team creation and look into not sanitizing the other ones for team admins